json/src/spec/params.rs

```diff
@@ -130,12 +135,36 @@ pub struct Params {
 	pub kip6_transition: Option<Uint>,
 }
 
+/// A contract that determines block gas limits can be configured either as a single address, or as a map, assigning
+/// to each starting block number the contract address that should be used from that block on.
```

Comment:
Maybe a dumb q but wouldn't the code be a bit simpler if the "single address" case was just a map with a single entry?

Reply:
Not a dumb question at all! I don't know of a sensible way to allow the user to choose from two configuration options: a single address and a map. If we simply remove the single address option, you would always need to define a map.

Reply:
> you would always need to define a map.

I think that is fine. I think simpler code is worth a little extra user-hassle because runtime bugs are very expensive to fix while a config problem is easy to fix and easy to document?

Reply:
@varasev, do we currently use a single block gas limit contract config option? If we don't, can we remove its support from the code?

Reply:
@vkomenda no, we don't. It's enough to only have a map.